pdaf-code/make.arch/linux_gfortran.h

```diff
@@ -58,5 +58,9 @@ MPI_INC =
 OBJ_MPI =
 
 # NetCDF (only required for Lorenz96)
-NC_LIB   =
-NC_INC   =
+NC_LIB   = -L/usr/lib/x86_64-linux-gnu -lnetcdff -Wl,-Bsymbolic-functions -flto=auto -ffat-lto-objects -Wl,-z,relro -Wl,-z,now -lnetcdf -lnetcdf -lm
+NC_INC   = -I/usr/include 
```

Comment:
This is currently platform specific. I think the following should work and be more portable?
```suggestion
NC_LIB   = $(nf-config --flibs)
NC_INC   = $(nf-config --includedir)
```